generators/entity-client/templates/angular/src/main/webapp/app/entities/entity.service.ts.ejs

```diff
@@ -45,91 +45,86 @@ export class <%= entityAngularName %>Service {
     <%_ } _%>
 
     constructor(private http: HttpClient) { }
-    <%_ if (entityAngularName.length <= 30) { _%>
 
+    <%_ if (entityAngularName.length <= 30) { _%>
```

Comment:
since we have prettier now, I think its time to get rid of the length based manual formatting